src/schema/partner_show_event.ts

```diff
@@ -1,8 +1,46 @@
-import date from "./fields/date"
-import { GraphQLString, GraphQLObjectType } from "graphql"
+import dateField, { date, DateSource } from "./fields/date"
+import { GraphQLString, GraphQLObjectType, GraphQLFieldConfig } from "graphql"
 import { ResolverContext } from "types/graphql"
 import { exhibitionPeriod } from "lib/date"
 
+const checkUserAgent = (userAgent): boolean =>
```

Comment:
Bit of a nit-pick, but I recommend a less generic name here - maybe hasOldEmissionUserAgent